Deprecate oVirt (#3724)
theforeman/foreman-documentation: guides/common/modules/proc_performing-post-upgrade-tasks.adoc

[id="performing-post-upgrade-tasks_{context}"]
= Performing post-upgrade tasks

> * Optional: If you are utilizing the oVirt compute resource, it's crucial to migrate your data by executing the `foreman-rake ovirt:drop` task.

Reviewer: Suggestions to address style requirements: Attribute for oVirt, avoiding contractions ("it's"), and keep it simple by using plain imperative voice ("migrate your data" rather than "it is crucial to migrate your data").

Suggestion to improve clarity: Where do I run it? On the server, right?

```suggestion
* Optional: If you are utilizing the {oVirt} compute resource, migrate your data by executing the `foreman-rake ovirt:drop` task on your {ProjectServer}.
```

> Remember that with the upcoming release of version 3.16, oVirt will be officially phased out, and this migration will be automatically managed during the upgrade process.

Reviewer: I wonder if we should avoid advertising specific changes for a particular release (because plans can change). May I suggest wording that's a bit more on the vague side, like "In a future version"?

```suggestion
In a future version, {oVirt} will be officially phased out, and this migration will be automatically managed during the upgrade process.
```
